server/routes/users.js

```diff
@@ -0,0 +1,27 @@
+var express = require("express");
+var router = express.Router();
+import models from '../models';
+
+router.post('/', (req, res) => {
+  let newUser = {
+    name: req.body.name,
+    email: req.body.email,
+    password: req.body.password
```

Comment:
I think that we need to encrypt the password before save it in the db. We can use the bcryptjs (async) to do it for us.

[https://www.npmjs.com/package/bcryptjs#usage---async](url)